dev-python/watchdog/watchdog-0.10.2.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7,8} )
+DISTUTILS_USE_SETUPTOOLS=rdepend
+
+inherit distutils-r1 eutils
+
+DESCRIPTION="Python API and shell utilities to monitor file system events"
+HOMEPAGE="https://github.com/gorakhargosh/watchdog"
+SRC_URI="https://github.com/gorakhargosh/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~x86"
+IUSE="test"
+
+CDEPEND="dev-python/pyyaml[${PYTHON_USEDEP}]"
+RDEPEND="${CDEPEND}
+	dev-python/argh[${PYTHON_USEDEP}]
+	dev-python/pathtools[${PYTHON_USEDEP}]"
+DEPEND="${CDEPEND}
+	test? ( dev-python/pytest-timeout[${PYTHON_USEDEP}] )"
+
+PATCHES=( ${FILESDIR}/${P}-drop-test-cov.patch )
+
+distutils_enable_tests pytest
+
+src_prepare() {
+	default
+	rm tox.ini || die
```

Comment:
can you try "deleting" `tox.ini` with the patch?

Reply:
I'll just close this as it's obsoleted.